### PR TITLE
Add a new Mixer API to call related places.

### PR DESCRIPTION
**Pull request by Spaceenter**

### CODE REVIEW

**Comment by Spaceenter** [comment]
Output from running examples/get_related_places/main.go:

$ go run main.go
Requesting { dcids:"geoId/06085" population_type:"Person" pvs:<property:"gender" value:"Female" > pvs:<property:"age" value:"Years21To64" > measured_property:"count" }
{
  "geoId/06085": [
    "geoId/12011",
    "geoId/35",
    "geoId/36061",
    "geoId/48029"
  ]
}
Request took: 1.470252135s

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Generate protobuf code and out.pb (used for cloud endpoints deployment).
 
     go run server/main.go \
       --bq_dataset=google.com:datcom-store-dev.dc_v3_clustered \
-      --bt_table=dc24 \
+      --bt_table=dc25 \
```
Comment (pradh) [inline README.md]: Please confirm with tjann on this?

File: proto/mixer.proto [hunk 2]
```diff
@@ -252,6 +253,44 @@ message GetPlacesInResponse {
...
+  // (Optional) Unit.
+  string unit = 9;
+
+  // The statistics type, like "measured", "median", etc...
+  string stat_type = 10;
+}
+
+// Response of GetRelatedPlaces request.
+message GetRelatedPlacesResponse {
+  // The JSON payload.
```
Comment (pradh) [inline proto/mixer.proto]: Will the response be described in docs?

File: proto/mixer.proto [hunk 3]
```diff
@@ -252,6 +253,44 @@ message GetPlacesInResponse {
...
+  // (Opitonal) Measurement qualifier.
+  string measurement_qualifier = 7;
+
+  // (Opitonal) Scaling factor.
+  string scaling_factor = 8;
+
+  // (Optional) Unit.
+  string unit = 9;
+
+  // The statistics type, like "measured", "median", etc...
```
Comment (pradh) [inline proto/mixer.proto]: is this the old style or new style ("measuredValue")?

Comment (pradh) [inline proto/mixer.proto]: Should we make it optional with default being "measuredValue"?

(in some other context, we were talking about it right?)

Comment (Spaceenter) [reply proto/mixer.proto]: It's "measured", "median"... etc.

File: README.md (hunk 1 above)
Comment (Spaceenter) [reply README.md]: Reverting this change. Defer to tjann on changing these.

File: proto/mixer.proto (hunk 3 above)
Comment (Spaceenter) [reply proto/mixer.proto]: Good idea - will change it when the BT has StatType (now BT doesn't include StatType).

**Pull request merged**